charts/elasticsearch/values.yaml

```diff
@@ -65,6 +65,12 @@ common:
 # by the ingest processors and the required resources, it may make sense to
 # have dedicated ingest nodes, that will only perform this specific task.
 client:
+
+  securityContext:
+    capabilities:
+      add:
+        - IPC_LOCK
+        - SYS_RESOURCE
```

Comment:
@manu / @danielhoherd - We can remove since elastic doesn't need any extra priv. There default is DROP.

Reply:
@pgvishnuram we need custom permissions for client other than drop that's the reason its present.

Reply:
@shmanu017 can we comment it out with a working `drop` example?

Reply:
Global helm values set `charts/elasticsearch/values.yaml` aren't getting overridden to null values in the test case to disable client security context to accept global security context.Since, we will not be facing this condition not adding test in this case.

Reply:
# Dont set any security context to es-client since it will need privileged access to run in openshift 
securityContext: []